scipy/sparse/linalg/tests/test_matfuncs.py

```diff
@@ -532,6 +532,24 @@ def test_matrix_input(self):
             B = expm(np.matrix(A))
         assert_allclose(B, B0)
 
+    def test_exp_sinch_overflow(self):
+        # Check overflow in intermediate steps is fixed (gh-11611)
```

Comment:
That's the gfortran-10 issue; should be: gh-11839 I think.

Otherwise looks solid

Reply:
Fixed.